static/js/containers/VideoDetailPage_test.js

```diff
@@ -120,6 +121,18 @@ describe('VideoDetailPage', () => {
     assert.isFalse(wrapper.find(".edit").exists());
   });
 
+  it('includes the download button that triggers dialog when the user has correct permissions', async () => {
```

Comment:
~~download button~~ -> `"save to dropbox" button` et al

Reply:
OK